CHANGELOG.md

```diff
@@ -1,35 +1,56 @@
+## 1.13.0 - 14 May 2021
+
+### Added
+
+* Use fixed port for .Net in compose files. [#2725](https://github.com/microsoft/vscode-docker/issues/2725)
+* Use latest version of dockerfile language server .NET apps will now scaffold using a non-root user. [#2865](https://github.com/microsoft/vscode-docker/issues/2865)
```

Comment:
I think this line 6 has two issues.
1. Two different things are being mentioned on the same line.
2. The link points to a specific problem that was fixed so perhaps it should be in the "Fixed" section instead?